src/ReflectionClosure.php

```diff
@@ -143,7 +135,7 @@ public function getCode()
                     if ($token[0] === T_FUNCTION || $token[0] === T_STATIC) {
                         $code .= $token[1];
                         $state = $token[0] === T_FUNCTION ? 'function' : 'static';
-                    } elseif ($fn && $token[0] === T_FN) {
+                    } elseif (\PHP_VERSION_ID >= 70400 && $token[0] === T_FN) {
```

Comment:
We should define `T_FN` before the class definition so we can remove the related version checks (`\PHP_VERSION_ID >= 70400`)
```php
defined('T_FN') || define('T_FN', -6);
```